src/history/index.test.js

```diff
@@ -0,0 +1,76 @@
+import fs from 'fs';
+import chai from 'chai';
+
+import { RAW_DIRECTORY, storeRaw, commitRaw, persistRaw } from './index.js';
+
+const expect = chai.expect;
+
+const SERVICE_PROVIDER_ID = 'test_service_provider';
+const POLICY_TYPE = 'terms_of_service';
+const FILE_CONTENT = 'ToS fixture data with UTF-8 çhãràčtęrs';
+const EXPECTED_FILE_PATH = `${RAW_DIRECTORY}/${SERVICE_PROVIDER_ID}/${POLICY_TYPE}.html`;
+
+
+describe('History', () => {
+  describe('#storeRaw', () => {
+    context("when service provider's directory already exist", () => {
+      after(() => {
+        fs.unlinkSync(EXPECTED_FILE_PATH);
```

Comment:
> I believe Mocha supports async `before` and `after` hooks as long as we return the promise, so we could use `unlink` rather than `unlinkSync` and accelerate the tests 🙂